microcosm_logging/formatters.py

```diff
@@ -26,6 +27,7 @@ def format(self, record):
         message = record.getMessage()
 
         extra = merge_record_extra(record, dict())
+        extra['_hostname'] = gethostname()
```

Comment:
Maybe `gethostbyname(gethostname)`?

It ought to give the IP, which is what we use in our inventory